flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
we use only-arches instead.

This is disappointing.

Reply:
Fixed.
And yeah, unfortunately PySide (Qt for Python) doesn't support ARM, disappointing indeed :(

Reply:
how come it doesn't.

Reply:
you should build pyside from source.

Reply:
PySide includes Qt, so building that properly would require expertise in C++ that I lack (and take very long to compile). Maybe you or someone else could provide it as a shared module? Regardless, letting the Flathub bot run with the only-arches flag removed resulted in a failed build on ARM.

Unfortunately I won't be able to support that platform, but if someone with more experience than I is able to make it work, I would gladly accept that.

Reply:
it if includes qt then you don't need the kde runtime.

you don't need expertise in C++, Just expertise in cut and paste.

Reply:
I use the KDE runtime because it provides the rest of the Qt integration (theme support etc.) so that this app integrates well into the desktop. I tried using the default freedesktop runtime, but themeing didn't work and the app looked broken. It's therefore a necessity.

As for building pyside from source, I tried several things for the last 2 hours and I haven't gotten past C++ build errors that I don't understand. I also tried looking for other projects using pyside on flathub, but couldn't find any building pyside from source, so that I could copy what they did. The matrix channel couldn't help either.

Maybe I'm just stupid and you can see the mistake right away, but here is what I tried:
```yaml
runtime: org.kde.Platform
runtime-version: '6.4'

...


  - name: pyside6
    buildsystem: cmake-ninja
    builddir: true
    build-options:
      env:
        CLANG_DIR: "/app/lib/cmake/Clang/"
    config-opts:
      - -DCMAKE_BUILD_TYPE=Release
      - -DBUILD_TESTS=OFF
    sources:
      - type: archive
        url: https://download.qt.io/official_releases/QtForPython/pyside6/PySide6-6.5.0-src/pyside-setup-everywhere-src-6.5.0.tar.xz
        sha256: 6ef53b29127267e381917e6f9399ce760edc0644cd5831988b1de72c9d583ab4
```